include/dlaf/solver/triangular/impl.h

```diff
@@ -330,6 +330,10 @@ template <Backend backend, Device device, class T>
 void Triangular<backend, device, T>::call_LLN(comm::CommunicatorGrid grid, blas::Diag diag, T alpha,
                                               Matrix<const T, device>& mat_a, Matrix<T, device>& mat_b) {
   using namespace triangular_lln;
+  using hpx::unwrapping;
+
+  using common::internal::vector;
+  using ConstTileType = typename Matrix<T, device>::ConstTileType;
```

Comment:
AFAICT they are superflous since #392, when the triangular solver has been migrated from using vector to Panel.

```suggestion
```